src/super_gradients/training/utils/checkpoint_utils.py

```diff
@@ -156,6 +161,1229 @@ def read_ckpt_state_dict(ckpt_path: str, device="cpu") -> Mapping[str, torch.Ten
         return state_dict
 
 
+class DefaultCheckpointSolver:
+    """
+    Default checkpoint solver class - has same behavior as PyTorch's default checkpoint solver.
+    """
```

Comment:
Would be great to have 2 words about this default behavior, not everyone would know what it means

Reply:
Fixed in cc81568ae886751206245efe29ade055f7b4ce9f

Reply:
Fixed in a117c77942e0867765794b248ce02de132c65895